test/mocha/10-meters.js

```diff
@@ -37,6 +39,41 @@ describe('meters.upsert()', () => {
     should.not.exist(err);
     res.meter.id.should.equal(`${meterService}/${data.meter.id}`);
   });
+  it('should return "false" if "result.n" is equal to 0', async () => {
```

Comment:
The test name shouldn't be specific to the implementation. What we're looking for here is that we get `false` when the update doesn't occur because there wasn't a match.

```suggestion
  it('should return "false" if no insert nor update occurs', async () => {
```